### PR TITLE
[msvc] install PDB files

### PR DESCRIPTION
**Pull request by matlo607**

### CODE REVIEW

File: googlemock/CMakeLists.txt [hunk 1]
```diff
@@ -145,6 +145,14 @@ if (gmock_build_tests)
   # 'make test' or ctest.
   enable_testing()
 
+  if (WIN32)
```
Comment (gennadiycivil) [inline googlemock/CMakeLists.txt]: This is making an assumption that on Windows the only tool people use is MSVC. Please check other conditions as well ( MinGW)

**Comment by matlo607** [comment]
Hello @gennadiycivil,
I am struggling to build googletest under MinGW, it seems unit tests have not been built for a long time.
Why didn't appveyor catch these issues earlier ?
I opened 2 issues (#1841 and #1842) and respectively 2 PR to address them (#1844 and #1843).

**Comment by gennadiycivil** [comment]
@matlo607 Thank you for looking into this. To answer your question why didn't appveyor catch these issues earlier - cant comment on this , don't know. We don't use MinGW internally and any support is community-based. 
If MinGW  is important for you As part of the work I would recommend also adding appropriate appveyor configuration so this does not happen again

**Comment by matlo607** [comment]
@gennadiycivil , I did a PR to enable the tests again #1853. However, since the build time can be quite long for one of gmock unittest, I am afraid that the build will timeout.

MinGW is not used internally in my company and I have no specific interest in it except that this PR be accepted. I fixed the unittests whereas they were broken and ensured the patch doesn't break them.
If you think the maturity of this patch is good, please merge it.

**Comment by gennadiycivil** [comment]
@matlo607 could you please address my previous comments (https://github.com/google/googletest/pull/1836#discussion_r217735374)

**Comment by matlo607** [comment]
It was already addressed, please have a look to the commit. I overwrote the history.
Maybe were you expecting a new commit ?

**Comment by gennadiycivil** [comment]
@matlo607 Thank you again for this contribution. Now lets get back to the original purpose of the fix.
The PR says "install PDB files". 
Could you please 
1. describe in better details why PDB files would be beneficial. (Remember that many developers do not work on Windows)
2. Please provide clear output before and after to demonstrate the changes

Thanks again

**Comment by matlo607** [comment]
@gennadiycivil, @paercebal : I rebased the branch on master and changed the output directory to /bin.

**Comment by gennadiycivil** [comment]
@matlo607 @paercebal I merged the changes. You may see that the PR is "closed" however the commits are merged in

**Pull request closed without merging**